sdk/react/upNext.js

```diff
@@ -101,15 +101,16 @@ var UpNext = React.createClass({
       var upNextWidth = ResponsiveDesignManager.makeResponsiveMultiplier(this.props.width, thumbnailWidth, [0.8,1], [520]);
       var countdown = this.renderCountdownTimer();
       var upNextImage = (
-      <TouchableHighlight style={[styles.thumbnailContainer, {width: upNextWidth}]}
```

Comment:
not a fan of this.  the entire image should be clickable.  why did you change this?